### DISCUSSION TITLE
Variable Url adress

### DISCUSSION THREAD

**Discussion by cm-slu** [grafana/grafana-infinity-datasource#421] [Q&A] [answered]
Hello, this is a subject probably already discussed but I have nothing on it.

I would like to be able to pass a url (ip address + port) as a variable.

Is that possible ?

**Comment by yesoreyeram** [comment]
Yes. Isn't that working?

**Comment by cm-slu** [comment]
I don't know yet because I haven't found how to do it ;)
I don't want to add parameters in the url but I would like the address to be a variable.

Ex

https://www.string_url/getUsers

**Reply by cm-slu** [reply]
sorry that was easy i didn't think about it.
$url\getUsers

Otherwise I take this opportunity to say that you did a good job for your addon bravo

**Comment by cm-slu** [comment, marked as answer]
So I have another question.
I managed to make a URL variable which I pass in every get.

If I have two different URLs and I want to choose ALL in the combobox, is it possible to loop through each URL and add up the result?

**Reply by yesoreyeram** [reply]
nope.. Clearly that is not the use case the plugins solves.

**Reply by cm-slu** [reply]
oh i'm sad.
I will find another solution so thank you very much
